File: salario/salario.java
import java.io.BufferedReader;
import java.io.IOException;
import java.io.InputStreamReader;

class Main {
    
    public static void main(String[] args) throws IOException {
        
        InputStreamReader ir = new InputStreamReader(System.in);
        BufferedReader in = new BufferedReader(ir);
        
        int number, horas;
        double salario_horas, salario;
        
        number = Integer.parseInt(in.readLine());
        horas = Integer.parseInt(in.readLine());
        salario_horas = Double.parseDouble(in.readLine());

        salario = horas * salario_horas;
        
        System.out.printf("NUMBER = %d\n", number);
        System.out.printf("SALARY = U$ %.2f\n", salario);
    }
    
}

File: salario/salario.c
#include <stdio.h>

int main(){

    int number, horas ;
    double salario_horas, salario;

    scanf("%d", &number);
    scanf("%d", &horas);
    scanf("%lf", &salario_horas);

    salario = horas * salario_horas;

    printf("NUMBER = %d\n", number);
    printf("SALARY = U$ %.2lf\n", salario);
    
    return 0;
}